### ISSUE TITLE
Import new issues and update issues 

### ISSUE THREAD

**Issue by viktorgr** [leovitch/redmine_importer#27]
It's posible to import new issues and to update exsinting issues from one file at the same time? Thank you

**Comment by leovitch** [comment]
No, it's not currently possible AFAIK.  I want to add a new option "skip existing issues" that would at least let you do this in two passes but I haven't gotten to it yet, my apologies.
